doc/scripts/build_bitmask_rst.py

```diff
@@ -0,0 +1,47 @@
+"""
+Dynamically build tables for the bitmasks.
+"""
+
+import os
+import time
+import numpy
+
+from pkg_resources import resource_filename
+from pypeit.utils import to_string, string_table
+
+from IPython import embed
+
+#-----------------------------------------------------------------------------
+
+def write_bitmask_table(obj, path):
+    bm = obj()
+    ofile = os.path.join(path, '{0}_table.rst'.format(obj.__name__.lower()))
+
+    data_table = numpy.empty((bm.nbits+1, 3), dtype=object)
+    data_table[0,:] = ['Bit Name', 'Bit Value', 'Description']
+    for i,k in enumerate(bm.bits.keys()):
+        data_table[i+1,0] = k
+        data_table[i+1,1] = to_string(bm.bits[k])
```

Comment:
Consider adding a `2**bm.bits[k]` column for those that
don't quite get the concept of a bit.  :)

Reply:
Done.